docs/contribute.rst

```diff
@@ -86,12 +84,11 @@ Any change you make should correspond with a documentation update.
 
 ::
 
+    $ pipenv install
```

Comment:
Since the docs requirements have moved to the general Pipfile, can we flip the order of the instructions on lines 83 and 84 above? We can also add a note that if they have already installed requirements, they do not need to do that again.

Reply:
Good call